core/src/test/java/org/elasticsearch/test/discovery/ClusterDiscoveryConfiguration.java

```diff
@@ -82,7 +82,7 @@ public UnicastZen(int numOfNodes, int numOfUnicastHosts, Settings extraSettings)
                 while (ordinals.size() != numOfUnicastHosts) {
                     ordinals.add(RandomizedTest.randomInt(numOfNodes - 1));
                 }
-                unicastHostOrdinals = Ints.toArray(ordinals);
+                unicastHostOrdinals = Integers.toArray(ordinals);
```

Comment:
Yes can we just use ordinals.stream().mapToInt(s -> s).toArray() ?

It would be great to remove the Integers.java completely then. We have too many source code files.